packages/app/templates/extensions/projects/flow_action/shopify.extension.toml.liquid

```diff
@@ -1,28 +1,19 @@
-name = "{{name}}"
+name = "{{ name }}"
+description = "Your description"
 type = "flow_action"
 
-[task]
-title = "extension-title"
-description = "extension-description"
-url = "https://runtime-endpoint.com"
-validation_url = "https://validation-url"
-custom_configuration_page_url = "https://custom-configuration-page-url"
-custom_configuration_page_preview_url = "https://custom-configuration-page-preview-url"
-return_type_ref = "SomeTypeFromSchema"
-schema = "./schema.graphql"
+[[extensions]]
+runtime_url = "https://url.com/api/execute"
```

Comment:
as mentioned before, `type` should also be part of the object in extensions array